app/uk/gov/hmrc/soletraderidentificationfrontend/views/templates/govuk_template.scala.html

```diff
@@ -73,11 +64,16 @@
     <script src='@controllers.routes.Assets.versioned("javascript/backLink.js")'></script>
```

Comment:
I don't think we need the other 4 scripts anymore with the new layout. Ran it without them and it all worked and no complaints from the console